src/templates/base/2019/index.html

```diff
@@ -24,6 +24,8 @@
   <link rel="stylesheet" href="/static/css/index.css">
 {% endblock %}
 
+{% block contributors %}{{ config.contributors.keys() | length }} {% endblock %}
```

Comment:
Creating a new block to pass the variable through. Elegant solution there. Well done!

Reply:
It's a little convoluted but works a treat and I think it makes the templates easier to translate - even if the logic is more difficult to understand.

Going to do same to fix your contributors issue to avoid reusing the same capitalised title.

Reply:
Done in https://github.com/HTTPArchive/almanac.httparchive.org/pull/685/commits/4064c1d1a633074c5ce5877fbad322b50b32b067. Let me know if I got the translation right.